terraform/test-environments/modules/chef_automate_install/main.tf

```diff
@@ -264,6 +264,8 @@ ${var.saml == "true" ? local.saml_config : ""}
 
 [elasticsearch.v1.sys.runtime]
   heapsize = "${var.enable_cloudwatch_metrics == "true" ? "16g" : "2g"}"
+[elasticsearch.v1.sys.cluster]
+  max_shards_per_node = ${var.enable_cloudwatch_metrics == "true" ? "1500" : "1000"}
```

Comment:
what about cloudwatch metrics is causing problems. Or is that an indicator for our perf environments

Reply:
The latter